src/main/java/com/aws/greengrass/componentmanager/converter/RecipeLoader.java

```diff
@@ -66,9 +67,9 @@ public static com.amazon.aws.iot.greengrass.component.common.ComponentRecipe par
                     .readValue(recipe, com.amazon.aws.iot.greengrass.component.common.ComponentRecipe.class);
         } catch (JsonProcessingException e) {
             // TODO: [P41216539]: move this to common model
-            throw new PackageLoadingException(
-                    String.format("Failed to parse recipe file content to contract model. Recipe file content: '%s'.",
-                            recipe), e);
+            throw new PackageLoadingException(String.format(
+                    "Failed to parse recipe file content to contract model. Recipe file content: '%s'.", recipe),
```

Comment:
should the full recipe contents be part of the exception message? that seems like it should just be logged

Reply:
will update